builtin/logical/nomad/path_config_access.go

```diff
@@ -0,0 +1,94 @@
+package nomad
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/vault/logical"
+	"github.com/hashicorp/vault/logical/framework"
+)
+
+func pathConfigAccess(b *backend) *framework.Path {
+	return &framework.Path{
+		Pattern: "config/access",
+		Fields: map[string]*framework.FieldSchema{
+			"address": &framework.FieldSchema{
+				Type:        framework.TypeString,
+				Description: "Nomad server address",
+			},
+
+			"token": &framework.FieldSchema{
+				Type:        framework.TypeString,
+				Description: "Token for API calls",
+			},
+		},
+
+		Callbacks: map[logical.Operation]framework.OperationFunc{
+			logical.ReadOperation:   b.pathConfigAccessRead,
+			logical.UpdateOperation: b.pathConfigAccessWrite,
+		},
+	}
+}
+
+func (b *backend) readConfigAccess(storage logical.Storage) (*accessConfig, error) {
+	entry, err := storage.Get("config/access")
+	if err != nil {
+		return nil, err
+	}
+	if entry == nil {
+		return nil, nil
+	}
+
+	conf := &accessConfig{}
+	if err := entry.DecodeJSON(conf); err != nil {
+		return nil, fmt.Errorf("error reading nomad access configuration: %s", err)
+	}
+
+	return conf, nil
+}
+
+func (b *backend) pathConfigAccessRead(
+	req *logical.Request, data *framework.FieldData) (*logical.Response, error) {
+	conf, err := b.readConfigAccess(req.Storage)
+	if err != nil {
+		return nil, err
+	}
+	if conf == nil {
+		return nil, nil
+	}
+
+	return &logical.Response{
+		Data: map[string]interface{}{
+			"address": conf.Address,
```

Comment:
Is it intentional that you are not returning the `token` here?

Reply:
Never mind. I see the test which explicitly ensures that the `token` is not in the response.

Reply:
I thought we had started to return secret information from the config endpoints, like tokens and passwords. Let me check what our current standard is.

Reply:
Just going to leave this as-is for now.

Reply:
@chrishoffman @vishalnayak Well... just thinking out loud but we could store/return the accessor of that token. That would potentially allow someone to check what's the accessor of Vault's Nomad Token and revoke it in case there's need to do so. That being said, Nomad allows to List accessors (with a name parameter to help identify them), as documented in https://www.nomadproject.io/api/acl-tokens.html#list-tokens, and basically we would have to ask the user to input the accessor when configuring Vault.

Reply:
In some backends we have started to return real values instead of obfuscated values, and warn on write that this is the behavior but we're not yet fully consistent. The main reason to do it is that it allows for exact checking of configuration state in automated ways (there's also a forensics argument). The flip side to that is that people probably shouldn't be holding those credentials in ways that allow configuration checking in the first place. Basically, there's no perfect answer here.

I think it's totally fine to just leave it out. It can always be added later.